META.github.template

```diff
@@ -0,0 +1,11 @@
+# JBUILDER_GEN
+
+package "js" (
+  description = "Deprecated. Use github-js directly"
+  requires = "cohttp-js"
+)
+
+package "unix" (
+  description = "Deprecated. Use github-unix directly"
+  requires = "github-unix"
+)
```

Comment:
but this doesn't hold if you are doing other breaking API changes of course, so a 3.0 is probably more appropriate to not have this.